forge-game/src/main/java/forge/game/card/Card.java

```diff
@@ -6893,6 +6895,7 @@ public void onCleanupPhase(final Player turn) {
         resetShieldCount();
         setBecameTargetThisTurn(false);
         setFoughtThisTurn(false);
+        setSurveilledThisTurn(false);
```

Comment:
currently this gets only called for battlefield cards
I guess we have to include graveyards now too? :/

Reply:
probably, i will do that later too